visualization.shadertoy/resources/language/resource.language.en_gb/strings.po

```diff
@@ -16,18 +16,18 @@ msgstr ""
 "Language: en_GB\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
-#. Addon summary used on Kodi's addon list and on websites
-#: visualization.shadertoy/addon.xml.in
 msgctxt "Addon Summary"
 msgid "Visualizations from Shadertoy"
 msgstr ""
 
-#. Addon description used on Kodi's addon list and on websites
-#: visualization.shadertoy/addon.xml.in
 msgctxt "Addon Description"
 msgid "This music visualization is based on Shadertoy based GPU programs."
 msgstr ""
 
+#. Addon summary used on Kodi's addon list and on websites
+#: visualization.shadertoy/addon.xml.in
+#. Addon description used on Kodi's addon list and on websites
+#: visualization.shadertoy/addon.xml.in
```

Comment:
@gade01 a idea why this was moved down? Comes it as there "msgctxt" without integer?
Better to leave comments there away?

Reply:
@AlwinEsch That doesn't look right.
I have no idea what happened, but perhaps @anxdpanic do? :)